Wit/WITRecorder.m

```diff
@@ -46,28 +47,45 @@ static void audioQueueInputCallback(void* data,
     int err;
     
     if (WIT_DEBUG) {
-        debug(@"Audio chunk %u/%u", (unsigned int)size, (unsigned int)buffer->mAudioDataBytesCapacity);
+        debug(@"Audio chunk %u/%u, numofpacdescriptions %d", (unsigned int)size, (unsigned int)buffer->mAudioDataBytesCapacity, numberPacketDescriptions);
     }
-
+    
     if (size > 0) {
         NSData* audio = [NSData dataWithBytes:bytes length:size];
         @autoreleasepool {
             WITRecorder* recorder = (__bridge WITRecorder*)data;
-            [recorder.delegate recorderGotChunk:audio];
-            if (recorder.vad != nil) {
+            if (recorder.delegate && recorder.isRecording == YES) {
```

Comment:
We don't need to check recorder.delegate, sending a message to nil is fine.

nit; just `(recorder.delegate && recorder.isRecording)`